src/it/enforce-bytecode-version-ignore-optional-dependencies/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>test</groupId>
+    <artifactId>enforce-bytecode-version-ignore-optional-dependencies</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <description>GitHub Issue #52 - ignoring optional dependencies </description>
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>@enforcerPluginVersion@</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>@project.groupId@</groupId>
+                        <artifactId>@project.artifactId@</artifactId>
+                        <version>@project.version@</version>
+                    </dependency>
+                </dependencies>
+                <configuration>
+                    <rules>
+                        <enforceBytecodeVersion>
+                            <maxJdkVersion>1.2</maxJdkVersion>
+                            <ignoredScopes>
+                                <!-- exclude junit/hamcrest -->
+                                <ignoredScope>test</ignoredScope>
+                            </ignoredScopes>
+                            <ignoreOptionals>true</ignoreOptionals>
+                        </enforceBytecodeVersion>
+                    </rules>
+                </configuration>
+                <executions>
+                    <execution>
+                        <phase>compile</phase>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.11</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <!-- this artifact contains classes targeting JDK 1.3 or newer -->
+            <groupId>javax.transaction</groupId>
+            <artifactId>jta</artifactId>
+            <version>1.1</version>

```

Comment:
FTR, I manually checked [this JAR](https://search.maven.org/remotecontent?filepath=javax/transaction/jta/1.1/jta-1.1.jar), just in case. So :+1: 

Reminder: `JDK 1.3 = 47 (0x2F hex)`

![image](https://user-images.githubusercontent.com/223853/49642945-ab98ae00-fa14-11e8-83cb-01abe4fd59e7.png)